registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessorTest.java

```diff
@@ -245,43 +249,64 @@ public void setup() throws Exception {
+		metainfo1.put(JsonConstant.CREATIONDATE,"2023-10-17T03:01:09.893");
+		MessageDTO object = packetValidateProcessor.process(messageDTO, stageName);
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(), Mockito.any(),
+				Mockito.any());
+		Assert.assertEquals(LocalDateTime.parse(metainfo1.get(JsonConstant.CREATIONDATE)), argument.getAllValues().get(0).getPacketCreateDateTime());
+		Assert.assertTrue(object.getIsValid());
+		Assert.assertFalse(object.getInternalError());
 	}
-	
+
 	@Test
-	public void PacketValidationSuccessTest() {
+	public void PacketValidationSuccessTestwithPacketCreatedDateTimeNull() throws PacketManagerException, ApisResourceAccessException, IOException, JsonProcessingException {
+		Map<String, String> metainfo = new HashMap<>();
+		metainfo.put(JsonConstant.CREATIONDATE,null);
+		Mockito.when(packetManagerService.getMetaInfo(any(), any(), any())).thenReturn(metainfo);
 		MessageDTO object = packetValidateProcessor.process(messageDTO, stageName);
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(), Mockito.any(),
+				Mockito.any());
+		assertEquals(metainfo.get(JsonConstant.CREATIONDATE), argument.getAllValues().get(0).getPacketCreateDateTime());
 		assertTrue(object.getIsValid());
 		assertFalse(object.getInternalError());
 	}
-	
+
 	@Test
-	public void PacketValidationFailureTest() throws PacketValidatorException, ApisResourceAccessException, JsonProcessingException, RegistrationProcessorCheckedException, IOException, PacketManagerException {
-		registrationStatusDto.setRetryCount(1);
-		Mockito.when(packetValidator.validate(any(), any(),any())).thenReturn(false);
+	public void PacketValidationSuccessTestwithPacketCreatedDateTimeInvalidFormat() throws PacketManagerException, ApisResourceAccessException, IOException, JsonProcessingException {
+		Map<String, String> metainfo = new HashMap<>();
+		metainfo.put(JsonConstant.CREATIONDATE,"2023-10-1703:01:09.893");
+		Mockito.when(packetManagerService.getMetaInfo(any(), any(), any())).thenReturn(metainfo);
 		MessageDTO object = packetValidateProcessor.process(messageDTO, stageName);
-		assertFalse(object.getIsValid());
-		assertFalse(object.getInternalError());
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(),any(),any());
+		assertEquals(null, argument.getAllValues().get(0).getPacketCreateDateTime());
 	}
-	
+
 	@Test
-	public void invalidSupervisorStatusTest() throws PacketValidatorException {
```

Comment:
Next time dont override the test cases ,please add the new test cases at the end of class